### DISCUSSION TITLE
Question: How to set pictures for Orgs&Team via API?

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#120862] [Enterprise]
I linke to change the avatars from Organizations and Teams to own pictures via API.
How to do this?

**Comment by queenofcorgis** [comment]
👋🏼 thanks for your patience @Timmmy-nlb ❤️ 

According to our [docs](https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#update-an-organization), it doesn’t appear that you can change the org avatar via the API.

**Reply by Timmmy-nlb** [reply]
Hallo and thank you for reply.
Then please, see it as a Feature Request.
